### DISCUSSION TITLE
All C calls should be moved to .c.v files.

### DISCUSSION THREAD

**Discussion by JalonSolov** [vlang/v#11741] [Ideas]
As of this moment, there are a LOT of `C.<whatever>` calls sprinkled throughout regular `.v` files in `vlib`.  This code needs to be separated so that `C.` calls are in `.c.v` files.

It would be really nice if the compiler also gave errors (warnings to start, perhaps, until everything has been separated properly) if any `C.` calls were to show up in plain `.v` files in the future.

**Comment by spytheman** [comment]
This should be best done gradually and carefully, perhaps by adding a cli flag, that turns on a stricter mode with more checks/warnings.

**Comment by spytheman** [comment]
Initial support for the warning is in #7195 .

**Comment by larpon** [comment]
We could also approach this like with #7047 - I think it would be quite easy to write a tool that will sniff them out and make a check list?
(`missdoc` could be modified for it also)
